src/i18n.ts

```diff
@@ -0,0 +1,14 @@
+import { notFound } from "next/navigation";
+import { getRequestConfig } from "next-intl/server";
+
+// Can be import from a shared config
+const locales = ["en", "ko"];
```

Comment:
maybe we could move this to `src/const/index.ts`
```suggestion
const LOCALES = ["en", "ko"];
```